x-pack/plugins/stack_alerts/public/alert_types/components/data_view_select_popover.tsx

```diff
@@ -102,18 +140,41 @@ export const DataViewSelectPopover: React.FunctionComponent<DataViewSelectPopove
           <DataViewsList
             dataViewsList={dataViewItems}
             onChangeDataView={(newId) => {
-              setSelectedDataViewId(newId);
-              const newTitle = dataViewItems?.find(({ id }) => id === newId)?.title;
-              if (newTitle) {
-                setSelectedTitle(newTitle);
-              }
-
               onSelectDataView(newId);
               closeDataViewPopover();
             }}
-            currentDataViewId={selectedDataViewId}
+            currentDataViewId={dataViewId}
           />
         </EuiFormRow>
+        {createDataView ? (
+          <EuiPopoverFooter paddingSize="none">
```

Comment:
@andreadelrio Updated styles for the button in the popover:

<img width="548" alt="Screenshot 2022-07-04 at 14 53 59" src="https://user-images.githubusercontent.com/1415710/177164855-a35aee74-882e-4bad-be86-c547d71b9837.png">

Reply:
Thanks @andreadelrio ! Updated:

<img width="516" alt="Screenshot 2022-07-06 at 10 53 23" src="https://user-images.githubusercontent.com/1415710/177515208-7428feff-2261-4cd4-8e94-80422a60ded3.png">

Is there anything else blocking?